reactive/etcd.py

```diff
@@ -13,19 +13,20 @@
 from charmhelpers.core.hookenv import config
 from charmhelpers.core.hookenv import unit_get
 from charmhelpers.core import host
-from charmhelpers.core.unitdata import kv
 from charmhelpers.core import templating
-from charmhelpers import fetch
 
 from etcd import EtcdHelper
+from subprocess import CalledProcessError
 
 @hook('config-changed')
 def remove_states():
+    # Matt - this is where it's not immutable :)
```

Comment:
Constants leave something to be desired really, as they should be configurable depending on the colocation needs